src/test/resources/suite-testng1.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="SuitaTestNG" verbose="1" >
+    <test name="AltTest1" >
+        <classes>
+            <class name="sorinfoca.altpachet.AltTest" />
+        </classes>
+    </test>
+
+    <test name="DemoTests">
+        <classes>
+            <class name="sorinfoca.Tests.MyTests"/>
+            <class name="sorinfoca.Tests.TestConfig"/>
```

Comment:
TestConfig doesn't have any test so you could remove it from this .xml file. Thanks!